ember.s

```diff
@@ -310,8 +345,7 @@ do_disk:
 error:
 	mov al, '!'
 	int i_putchar
-	cli
-	hlt
+	jmp shell
```

Comment:
What procedure did you use to test this? Also, since this is inside an interrupt, it seems that it will be leaking stack space.